articles/libraries/auth0-swift/index.md

```diff
@@ -77,7 +79,30 @@ Auth0
    }
 ```
 
-#### Passwordless Login
+### Passwordless Login
+
+Logging in with a Passwordless is slightly different. Passwordless can be done via email or via SMS, and either by sending the user a code, or sending them a link which contains a code. 
+
+#### Passwordless Parameters
+
+As you can see, Passwordless authentication can be started with a variety of different parameters. 
+
+```
+.startPasswordless(email/phoneNumber: String, type: String, connection: String)
```

Comment:
Might look cleaner to break this out into the two methods of `.startPasswordless(email` and `.startPasswordless(phoneNumber`